apps/webapp/next.config.js

```diff
@@ -1,18 +1,32 @@
-module.exports = {
-  reactStrictMode: true,
-  transpilePackages: ['ui'],
-  images: {
-    domains: ['avatar.vercel.sh'],
-  },
-  webpack: config => {
-    config.module.rules.push({
-      test: /\.mp4$/,
-      type: 'asset/resource',
-      generator: {
-        filename: 'videos/[hash][ext][query]',
-      },
-    });
+module.exports = () => {
+  const isDev = process.env.NODE_ENV !== 'production';
+  const prodConfig = !isDev ? { output: 'export' } : {};
```

Comment:
Would do this a bit differently:

```javascript
  const isProd = process.env.NODE_ENV === 'production';

  /**
   * @type {import('next').NextConfig}
   */
  const nextJsConfig = {
    output: isProd ? 'export' : undefined,
    // and so on
  };

  return nextJsConfig;
```
Adding the type will help ensure we don't add the wrong config.